### PR TITLE
 The homework for Js-3 week-1 added

### PR DESCRIPTION
**Pull request by basafilm**

### CODE REVIEW

File: javascript/javascript3/week1/homeWork-2/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        return discription.description
+    })
+//temperature
+    let teperature =parseFloat((resulte.main.temp-273.15).toFixed(2));
+    let feels =parseFloat((resulte.main.feels_like-273.15).toFixed(2));
+// Sun rise and set
+   let sunRise = new Date(resulte.sys.sunrise * 1000).toGMTString().slice(-12, -7);
+   let sunSet = new Date(resulte.sys.sunset * 1000).toGMTString().slice(-12, -7);
+// wind
+   let winde = parseFloat((resulte.wind.speed* 1.609344).toFixed(2));
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/homeWork-2/index.js:24]: wind

File: javascript/javascript3/week1/homeWork-2/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+// Weather app
+//efe28836fdf80aeac319e828033d2561
+
+function myFunction(){
+    const weatheUrl=  `https://api.openweathermap.org/data/2.5/weather?q=copenhagen,dk&appid=efe28836fdf80aeac319e828033d2561`
+fetch(weatheUrl)
+.then(response => response.json())
+.then(resulte=>{
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/homeWork-2/index.js:8]: don't put everything inside `.then()`, use callback functions.

File: javascript/javascript3/week1/Giphy api/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    //console.log(searchUrl)
+    fetch(searchUrl)
+    .then(response=> response.json())
+    .then(resulte =>{
+      let fig =document.createElement('figure');  
+      let image =document.createElement('img');
+      let caption= document.createElement('figcaption');
+      image.src = resulte.data[1].images.downsized.url;
+      image.alt = resulte.data.title;
+      image.style.width= '480px';
```
[the remark this answers is not in the document]
Comment (basafilm) [reply javascript/javascript3/week1/Giphy api/index.js:19]: ok. thanks

File: javascript/javascript3/week1/Giphy api/index.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>Document</title>
+</head>
```
[the remark this answers is not in the document]
Comment (basafilm) [reply javascript/javascript3/week1/Giphy api/index.html:8]: Thank you for feedback . You are right. My attention was only on JS.

File: javascript/javascript3/week1/Giphy api/index.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+
+let myKey="n3YDsd2r2uEPaLSQTTUbEYv3kr4uvA9I"
+window.addEventListener('DOMContentLoaded', find) 
+function find(){
```
[the remark this answers is not in the document]
Comment (basafilm) [reply javascript/javascript3/week1/Giphy api/index.js:4]: ok

**Pull request merged**